hotsos/core/plugins/kernel/kernlog/calltrace.py

```diff
@@ -206,6 +206,58 @@ def fields(self):
                 'local_pcp', 'free_cma']
 
 
+class BcacheTraceType(TraceTypeBase):
+
+    def __init__(self):
+        self._search_def = None
+        self.generics = []
```

Comment:
can you rename "generics" to something else e.g. "traces"

Reply:
I've called it deadlocks